src/app.ts

```diff
@@ -96,6 +103,64 @@ function withPandaAuth(
 		});
 }
 
+function getUI() {
+	return `
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Image URL Signing Service - UI</title>
+	</head>
+	<body>
+		<h1>Image URL Signing Service - UI</h1>
+		<p>You are logged in</p>
+		<p>Provide an image URL from the grid to get a URL signed by the Fastly image resizing service that can be used in production.</p>
+		<form action="/signed-image-url" method="GET">
+
+			<label for="url">Image URL:</label><br>
+			<input type="text" id="url" name="url" value=""><br><br>
+
+			<label for="width">Width:</label><br>
+			<input type="number" id="width" name="width" value="400"><br><br>
+
+			<label for="height">Height:</label><br>
+			<input type="number" id="height" name="height"><br><br>
+
+			<label for="height">Quality (value between 0 and 100):</label><br>
+			<input type="number" id="quality" name="quality" value="75"><br><br>
+
+			<input type="submit" value="Submit">
+		  </form>
+	</body>
+</html>
+`;
+}
+
+function getRedirectResponse(req: express.Request) {
+	const userHost = req.get('host') ?? '';
+	let returnUrl = `${req.protocol}://${userHost}${req.originalUrl}`;
+	if (returnUrl.includes('localhost')) {
+		returnUrl =
+			'https://image-url-signing-service.local.dev-gutools.co.uk/';
+	}
```

Comment:
I'd vote for not including this exception in the application, just to keep things simple. What do you think?

Reply:
I agree as if the redirect works then you should be accessing this from that endpoint regardless.